frank-flow/src/frontend/src/app/flow/node/nodes/node.model.ts

```diff
@@ -30,6 +32,7 @@ export class Node {
     this.left = options.left;
     this.attributes = options.attributes;
     this.senders = options.senders;
```

Comment:
Could you please remove the old "senders" code for this.

Also in xml-to-structure worker